CHANGELOG.md

```diff
@@ -6,3 +6,21 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
+
+## Version 0.2.0 - 2019-02-07
```

Comment:
Check how https://github.com/ArkEcosystem/core/blob/develop/CHANGELOG.md formats and links the versions.